server/@types/CvlUserDetails.ts

```diff
@@ -36,4 +36,4 @@ class CvlUserDetails {
   emailAddress?: string
 }
 
-export = CvlUserDetails
+export type User = Express.User & CvlUserDetails
```

Comment:
I wonder about this bit? The `CvlUserDetails` was created to hold the values we pull from Nomis, Auth and Delius based on the user auth source, values useful to hold in their session (e.g. staff ids, teams, ldu, caseload etc). Do we need to pass this structure, enhanced with the express User details to each of the API clients? Would it be clearer to have a new type which only contains the `username` and `user token` for the clear purpose for API clients to decide themselves which value they will use for each call?

Reply:
Sorry to be a pain - I know you've worked hard on this (69 files and counting on this PR!), and its all great stuff which will help - in fact, I think we could share this work for other services to follow as a template. Its really core to the service and the structure of how routes, services, API client and rest clients are wired together, so really important to make this as clear and simple as possible.

Reply:
By the way, please argue the case if you think this IS the right place for it!

Reply:
Change of mind - lets leave it! Its convenient, available and contains everything services need. A minimal object could be introduced in future, just containing `{ username: xxx, token: xxxx, displayName: xxxx }` which is required by services and API clients.

Reply:
Yes probably should have a specific type, would include `prisonCaseload` i think too though